cmd/lifecycle/exporter.go

```diff
@@ -259,7 +259,9 @@ func initDaemonImage(imagName string, runImageRef string, analyzedMD lifecycle.A
 		opts = append(opts, local.WithPreviousImage(analyzedMD.Image.Reference))
 	}
 
-	appImage, err := local.NewImage(
+	var appImage imgutil.Image
+	var err error
+	appImage, err = local.NewImage(
```

Comment:
I'm a little confused by this diff - this func is now called `initDaemonAppImage` on main: https://github.com/buildpacks/lifecycle/blob/9bb33743a01758f15d6135005adf395dca7fe13b/cmd/lifecycle/exporter.go#L263

Reply:
This diff is due to some changes in `imgutil` that don't relate to my changes. I just had to bump `imgutil` locally in order to test my code and I got an error on this assignment and it was solved by line 262.
Anyway I'll go ahead and rebase my branch.

Reply:
I've pushed updated code after rebasing my branch.
I'm going to resolve this conversation.
Please reopen it if something is still unclear.